### PR TITLE
fix for selective seq length test with batch size 1

### PR DESCRIPTION
**Pull request by kalyanjk**
fix for selective seq length test with batch size 1

### CODE REVIEW

**Review by ssarkar2 — COMMENTED** [review]
Can you provide more context in the ticket description why this is needed?

File: examples/text-generation/utils.py [hunk 1]
```diff
@@ -460,10 +460,28 @@ def setup_generation_config(args, model, assistant_model, tokenizer):
 
     return generation_config
 
+def exclude_hpu_graph_configs(args):
+    # Excluded configs for batch size 1 for hpu graph
+    if args.batch_size == 1 and args.limit_hpu_graphs:
+        if "falcon-180B" in args.model_name_or_path or \
+           "falcon-180b" in args.model_name_or_path:
```
Comment (ssarkar2) [inline examples/text-generation/utils.py:467]: could be:
"falcon-180b" in args.model_name_or_path.lower()

**Review by ssarkar2 — COMMENTED** [review]
ok, this seems to be a OHF port:

https://github.com/HabanaAI/optimum-habana-fork/commit/175256e12df2e650dae5b21a2b8022f542acb088

**Review by ssarkar2 — APPROVED** [review]

**Review by regisss — APPROVED** [review]

**Comment by regisss** [comment]
@kalyanjk Please run `make style`

**Pull request closed without merging**